entrypoint.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash -e
+
+if [[ -z $OWNER ]]; then
+  OWNER=lystable
+fi
+
+if [[ -z $REPO ]]; then
+  echo 'You must set the REPO environment variable'
+  exit 1
+fi
+
+if [[ ! -d /git/$REPO ]]; then
+  # Clone the repo if it's not cloned
+  echo "Repo not found, cloning now..."
+  git clone -b develop https://github.com/$OWNER/$REPO.git /git/$REPO > /dev/null
```

Comment:
If we want to make sure names from outside the `lystable` namespace don’t clash, we should record the owner name in the cache path. As in 
<pre>
git clone -b develop https://github.com/$OWNER/$REPO.git /git/<b>$OWNER/</b>$REPO > /dev/null
</pre>

`git clone` creates the target path recursively, like `mkdir -p` 😋